docs/sources/examples/mongodb.md

```diff
@@ -133,19 +136,19 @@ as daemon process(es).
 
     # Basic way
     # Usage: docker run --name <name for container> -d <user-name>/<repository>
-    $ docker run --name mongo_instance_001 -d my/repo
+    $ docker run -p 27017:27017 --name mongo_instance_001 -d my/repo
```

Comment:
Port 27017 is the default port that the mongodb process will use, unless its started with the '--port xxxxx' parameter. I have found that unless I pass the '-p 27017:27017' parameter on the run command, then I cannot connect from outside the container to the mongodb process inside the container. Perhaps this is a mis-understanding on my part.